src/PaperG/FirehoundBlob/Facebook/FacebookCreative.php

```diff
@@ -69,6 +69,18 @@ public function getType()
         return $this->type;
     }
 
+    public function isValid()
+    {
+        $creativeData = $this->getObjects();
+        $creativeType = $this->getType();
+
+        if (empty($creativeType) || empty($creativeData)) {
```

Comment:
Up to you, but you another way to write this is:

`return empty($creativeType) || empty($creativeData)`